File: src/main/java/br/com/alfser/projeto/pagamentos/config/SwaggerConfig.java
package br.com.alfser.projeto.pagamentos.config;

import io.swagger.v3.oas.models.OpenAPI;
import io.swagger.v3.oas.models.info.Info;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class SwaggerConfig {

    @Bean
    public OpenAPI openAPIDoc() {
        return new OpenAPI()
                .info(new Info().title("API de Pagamento")
                        .description(
                                "API de Genenciamento de Pagamentos.")
                        .version("v1"));
    }

}

File: processa-pagamentos/src/main/kotlin/br/com/alfser/projeto/processa_pagamentos/config/SwaggerConfig.kt
package br.com.alfser.projeto.processa_pagamentos.config

import io.swagger.v3.oas.models.OpenAPI
import io.swagger.v3.oas.models.info.Info
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class SwaggerConfig {
    @Bean
    fun openAPIDoc(): OpenAPI {
        return OpenAPI()
            .info(
                Info().title("Serviço de Processamento de Pagamento")
                    .description(
                        "Serviço de Processamento de Pagamentos."
                    )
                    .version("v1")
            )
    }
}